### PR TITLE
Improved: Set timezone from luxon settings instead of moment(#2dewerv)

### PR DESCRIPTION
**Pull request by shashwatbangar**
### Related Issues

Closes #
https://github.com/hotwax/picking/commit/c6aaf633bedbb408c20d326e1358c842aa0a5339

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [x] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,6 +51,7 @@
     "eslint-plugin-vue": "^7.0.0-0",
     "typescript": "~3.9.3",
     "vue-cli-plugin-i18n": "^1.0.1",
-    "vue-jest": "^5.0.0-0"
+    "vue-jest": "^5.0.0-0",
+    "vue-loader-v16": "^16.0.0-beta.5.4"
```
Comment (disha1202) [inline package.json:55]: Revert unwanted changes.

File: src/main.ts [hunk 2]
```diff
@@ -42,19 +41,22 @@ app.config.globalProperties.$filters = {
   formatDate(value: any, inFormat?: string, outFormat?: string) {
     // TODO Use Loxon instead
```
Comment (disha1202) [inline src/main.ts:42]: We can remove this TODO now.

**Review by disha1202 — CHANGES_REQUESTED** [review]

File: src/main.ts (hunk 2 above)
Comment (shashwatbangar) [reply src/main.ts:42]: Sure. Fixed it.

File: package.json (hunk 1 above)
Comment (shashwatbangar) [reply package.json:55]: Done.

**Comment by k2maan** [comment]
Closing as already implemented.

**Pull request closed without merging**